proto/cometbft/p2p/v1/conn.proto

```
@@ -0,0 +1,38 @@
+syntax = "proto3";
+package cometbft.p2p.v1;
+
+option go_package = "github.com/cometbft/cometbft/api/cometbft/p2p/v1";
+
+import "gogoproto/gogo.proto";
```

Buf (1.47.2)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)